addons/mail/static/src/models/thread/thread.js

```diff
@@ -446,7 +446,7 @@ function factory(dependencies) {
                         isMobile: device.isMobile,
                     }),
                 },
-            });
+            }, { shadow: true });
```

Comment:
I would revert this one, this is a direct user action for which we don't handle "loading" ourselves.